### PR TITLE
fix: exclude stem samples for QML waveform

### PR DESCRIPTION
**Pull request by acolombier**
Fixes the same issue than #13472 on QML

### CODE REVIEW

File: src/qml/qmlplayerproxy.cpp [hunk 1]
```diff
@@ -215,8 +215,26 @@ void QmlPlayerProxy::slotWaveformChanged() {
...
+
+    const WaveformData* data = pWaveform->data();
+    // Make a copy of the waveform data, stripping the stems portion. Note that the datasize is
+    // different from the texture size -- we want the full texture size so the upload works. See
+    // m_data in waveform/waveform.h.
+    if (m_waveformData.size() == 0 ||
+            static_cast<int>(m_waveformData.size()) !=
+                    pWaveform->getTextureSize()) {
+        m_waveformData.resize(pWaveform->getTextureSize());
+    }
```
Comment (Swiftb0y) [inline src/qml/qmlplayerproxy.cpp:227]: Why not call resize unconditionally? I don't see why this `if` is needed.

Comment (acolombier) [reply src/qml/qmlplayerproxy.cpp:227]: I just copied the patch that was merge for the legacy UI. Make sense to remove it tho!

Comment (Swiftb0y) [reply src/qml/qmlplayerproxy.cpp:227]: yeah. I'm not even sure how safe the condition is technically.

**Review by Swiftb0y — APPROVED** [review]
LGTM. Thank you

**Pull request merged**